homeassistant/components/sensor/teksavvy.py

```diff
@@ -18,6 +18,11 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 from homeassistant.util import Throttle
+# Python 3.4 support
+try:
+    from json.decoder import JSONDecodeError
```

Comment:
Use ValueError. JSONDecodeError is new in Python 3.4.

(or wait 2 weeks and we drop Py34 support)

Reply:
Dropped to `ValueError` only.